### PR TITLE
Applying flowtype to the components

### PR DESCRIPTION
**Pull request by joaoeffting**
Applued Flowtype to the components. It was a simple implementation because the components are simple yet.

### CODE REVIEW

File: src/components/BackgroundImage.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 import Globals from '../Globals';
 
+import type { Children } from 'react';
+
+type Props = {
+  children?: Children
+}
+
 class BackgroundImage extends Component {
+  props: Props;
```
Comment (sibelius) [inline src/components/BackgroundImage.js]: do you like this https://flow.org/en/docs/frameworks/react/#toc-explicitly-specifying-react-component-generics?

File: src/screens/Schedule.js [hunk 2]
```diff
@@ -1,47 +1,52 @@
...
 import Globals from '../Globals';
 
+type SchenduleType = {
+  time: string,
+  title: string,
+  description?: string,
+  imageUrl?: string
+}
+
+type Schendules = Array<SchenduleType>;
```
Comment (sibelius) [inline src/screens/Schedule.js]: typo: Schedules

File: src/components/BackgroundImage.js (hunk 1 above)
Comment (joaoeffting) [inline src/components/BackgroundImage.js]: Yeah .. I think it's better ... tonight I'm gonna adjust it. Tks!! I've never used props/state like this ...

**Comment by sibelius** [comment]
great job 💯

**Pull request merged**